packit_service/config.py

```diff
@@ -188,3 +190,24 @@ def get_package_config_from_repo(
                     logger.debug(f"Created issue for invalid packit config: {i.url}")
             raise ex
         return package_config
+
+
+class PagurePackageConfigGetter:
```

Comment:
What is the purpose if this class.

I both see `packit.yaml` and `*.spec` here? How do these 2 come together? Is spec-file the same as a config (packit.yaml) file?

Reply:
Please in checkpackit/config/package_config.py check get_package_config_from_repo() for implmentation this one is simplified version because of limited Pagure api capabilities.

Because of it is github api dependent, is imposible to use it for Pagure. Didnt want to spread "mess" in other repos becuase:
1. we need something what is working
2. there other open question for arch meeting which can possible affect how to implement this in clean way